src/parser/parser.ts

```diff
@@ -299,10 +311,15 @@ export function parse(
       if (rules.noCommaOperator) lintFlags |= LinterFlags.NoCommaOperator;
       if (rules.noDebugger) lintFlags |= LinterFlags.NoDebugger;
       if (rules.noDelete) lintFlags |= LinterFlags.NoDelete;
-      if (rules.noEmptyBlocks) lintFlags |= LinterFlags.NoEmptyBlocks;
+      if (rules.NoEmpty) lintFlags |= LinterFlags.NoEmpty;
```

Comment:
@aladdin-add Look at this rules. They are okay vs ESLint? It's the same kind of rules except that I skipped all the overhead, AST walking and performance regression found in ESLint.